terrasnek/projects.py

```diff
@@ -59,7 +59,7 @@ def list(self, page=None, page_size=None, filters=None):
             <https://developer.hashicorp.com/terraform/cloud-docs/api-docs/projects#query-parameters>`__
         """
         return self._list(\
-            self._org_api_v2_base_url, page=page, page_size=page_size, filters=filters)
+            self._org_api_v2_base_url, query=None, filters=filters, page=page, page_size=page_size)
```

Comment:
You want `query=query` here, otherwise I'll approve this.

Reply:
I've seen you've already applied, thank you!